### DISCUSSION TITLE
Welcome to baram Discussions!

### DISCUSSION THREAD

**Discussion by jiban** [nextfoam/baram#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by belosi57** [comment]
I faced with BARAM few days ago and I wanted to learn it. I installed BARAM v.23.3.5 for windows. When I launch BARAMFLOW only incompressible flow is available. Is not possible to simulate bouyancy flows?
Thank you

**Comment by jiban** [comment]
Hi, @belosi57 ,
Of course, BaramFlow is able to simulate buoyant flow.
"Incompressible Flow" in the case wizard means what is not "Compressible Flow", where "Compressible Flow" means a flow which has near-sonic speed.
In short, you can simulate buoyant flow with the *incompressible flow* option.
Thank you for your interest in Baram.

**Comment by belosi57** [comment]
Thank you for your quickly replay. Is there any english written BARAM guide?

**Reply by kdg5424** [reply]
Hi @belosi57.
I'm sorry for we are not providing english BARAM guide now.
We will update as English version as soon as possible.

In Right now, We are providing BARAM tutorial Youtube videos with english subtitles.
Why don't you watch it?
we recommend you visit below link and watch the videos.
Youtube Link : https://www.youtube.com/@nextfoam4326/videos
